[{"ContainerToolsException.Message":"Visual Studio Container Tools requires Docker to be running before building, debugging or running a containerized project.  For more info, please see: https://aka.ms/DockerToolsTroubleshooting","ContainerToolsException.ErrorCode":"","ContainerToolsException.StackTrace":"","ContainerToolsException.HResult":"-2146233088","ContainerToolsException.InnerException":"Microsoft.VisualStudio.Containers.Tools.Common.ContainerToolsException: error during connect: this error may indicate that the docker daemon is not running: Get \"http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/json?filters=%7B%22label%22%3A%7B%22com.docker.compose.config-hash%22%3Atrue%2C%22com.docker.compose.project%3Ddockercompose16582280088143936847%22%3Atrue%7D%7D\": open //./pipe/docker_engine: The system cannot find the file specified.\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<ExecuteAsync>d__13.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:linha 253\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerComposeClient.<DownAsync>d__8.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerComposeClient.cs:linha 104\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.DockerWorkspace.<ComposeDownContainersAsync>d__32.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker\\DockerWorkspace.cs:linha 424\r\n--- Fim do rastreamento de pilha do local anterior onde a exceção foi gerada ---\r\n   em System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)\r\n   em System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)\r\n   em Microsoft.Docker.BuildTasks.DockerBuildTask.<ExecuteAsync>d__34.MoveNext() na D:\\a\\_work\\1\\s\\src\\Microsoft.Docker.BuildTasks\\DockerBuildTask.cs:linha 82\r\nCódigo de erro: DT1001\r\nGravidade: Error\r\nLinha de comando: docker-compose  -f \"C:\\Users\\Yago\\OneDrive\\Desktop\\desafioDotNet\\docker-compose.yml\" -f \"C:\\Users\\Yago\\OneDrive\\Desktop\\desafioDotNet\\docker-compose.override.yml\" -f \"C:\\Users\\Yago\\OneDrive\\Desktop\\desafioDotNet\\obj\\Docker\\docker-compose.vs.release.partial.g.yml\" -p dockercompose16582280088143936847 --ansi never kill\r\nCódigo de saída do comando: 1\r\nSaída de comando: \r\nErro de comando: error during connect: this error may indicate that the docker daemon is not running: Get \"http://%2F%2F.%2Fpipe%2Fdocker_engine/v1.24/containers/json?filters=%7B%22label%22%3A%7B%22com.docker.compose.config-hash%22%3Atrue%2C%22com.docker.compose.project%3Ddockercompose16582280088143936847%22%3Atrue%7D%7D\": open //./pipe/docker_engine: The system cannot find the file specified.\r\n"}]